zeroforcing/fastqueue.pyx

```diff
@@ -0,0 +1,34 @@
+# Disabling bounds checking in this class sounds like it would be perfect here(?)
+cdef class FastQueueForBFS:
+    def __init__(self, max_priority):
+        self.max_possible_priority = max_priority
+        self.smallest_nonempty_priority = UINT_MAX
+        
```

Comment:
Maybe could use [conditional compilation](https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#conditional-compilation) (pretty much ifdefs) to include additional profiling information if a flag is enabled?